src/Plugin.php

```diff
@@ -242,4 +250,29 @@ public static function getBasePath() {
     return dirname(__DIR__);
   }
 
+  /**
+   * @implements graphql_register_types
+   */
+  public static function graphql_register_types() {
+    register_graphql_connection(
+      [
+        'fromType' => 'ProductVariation',
+        'toType' => 'MediaItem',
+        'fromFieldName' => 'galleryImages',
+        'resolve' => function ($source, array $args, $context, $info) {
+          $resolver = new \WPGraphQL\Data\Connection\PostObjectConnectionResolver($source, $args, $context, $info, 'attachment');
+          $resolver->set_query_arg('post_type', 'attachment');
+          $resolver->set_query_arg('post__in', Woocommerce::getVariationGalleryImages($source->databaseId) ?: ['0']);
+
+          // Change default ordering.
+          if (!in_array('orderby', array_keys($resolver->get_query_args()), TRUE)) {
```

Comment:
This looks like the slowest alternative to an isset(). Am I mistaken?

Please either use isset() or array_key_exists(). The latter is only needed if the value can be null.

Reply:
Good catch. I didn't analyze this part as I took from the example, but I agree 100%. The slowest `isset` ever.  Should we report it upstream?